agent/sshd/server.go

```diff
@@ -9,19 +9,17 @@ import (
 	"sync"
 	"time"
 
-	sshserver "github.com/gliderlabs/ssh"
-	"github.com/shellhub-io/shellhub/agent/pkg/osauth"
-	"github.com/sirupsen/logrus"
-)
-import (
 	"crypto"
```

Comment:
This belongs to the original commit. Not this rework patch right?

You could squash this as well, if preferred.

Reply:
done